packages/app/src/components/popup/token-assets.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { Flex, Text, Box, BoxProps, color } from '@stacks/ui';
+import type { AddressBalanceResponse } from '@blockstack/stacks-blockchain-api-types';
+import { stacksValue } from '@common/stacks-utils';
+import { AssetRow } from './asset-row';
+
+const NoTokens: React.FC<BoxProps> = props => (
+  <Box width="100%" py="extra-loose" textAlign="center" {...props}>
+    <Text color={color('text-caption')} fontSize={1} display="block" mb="extra-tight">
+      You don't own any tokens.
+    </Text>
+    <Text
+      as="a"
+      href="https://binance.com"
```

Comment:
In wallet I'm linking to CMC, probably best not to sponsor a given exchange

Reply:
So instead, we link to a different site owned by Binance 😛 but I agree, CMC is better, I've changed it.